config/locales/de.yml

```diff
@@ -0,0 +1,204 @@
+de:
+  title: "RubyGems.org"
+  subtitle: "Deine Community des Gem-Hostingservices"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Zugriff verweigert. Bitte melden Sie in https://rubygems.org mit einem Konto an"
```

Comment:
This sounds weird. First of all, the `subtitle` above uses "Deine" (colloquial) but here you use "Sie" (polite). Should be consistent. But the grammer is wrong too. Something like "Bitte melden Sie sich unter ... an" sounds more natural to me.